content/en/news/releases/1.12.x/announcing-1.12.6/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.12.6
+linktitle: 1.12.6
+subtitle: Patch Release
+description: Istio 1.12.6 patch release.
+publishdate: 2022-03-31
+release: 1.12.6
+aliases:
+    - /news/announcing-1.12.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.5 and Istio 1.12.6
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue where executing `istioctl upgrade` on 1.12 would result in webhook overlap errors.
+  ([Issue #37908](https://github.com/istio/istio/issues/37908))
+
+- **Fixed** an issue that caused TCP calls to still be logged after disabling the access logging through the Telemetry API.
```

Comment:
I think this will need `([Issue #37723](https://github.com/istio/istio/issues/37723))` but the issue hasn't been closed out for some reason?

Reply:
https://github.com/istio/istio/pull/38032 says it fixes it. The PR was never linked though.